src/components/customAGGrid/cell-renderers.tsx

```
@@ -5,7 +5,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import { Box, Checkbox, Theme, Tooltip } from '@mui/material';
+import { Box, Checkbox, Theme } from '@mui/material';
+import { CustomTooltip } from '../tooltip/CustomTooltip';
```

ESLint

[error] `../tooltip/CustomTooltip` import should occur after import of `react-intl`

(import-x/order)